src/test/java/httpserver/HttpPresenterTest.java

```diff
@@ -0,0 +1,19 @@
+package httpserver;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HttpPresenterTest {
+    @Test
+    void itFormatsAnHttpResponse() {
+        HttpPresenter presenter = new HttpPresenter();
+        String responseLine = "HTTP/1.1 405 Method Not Allowed" + System.lineSeparator();
+        String header = "Allow: HEAD, OPTIONS" + System.lineSeparator();
+        String blankLine = System.lineSeparator();
+        String body = "";
```

Comment:
Not sure what version of Java you are using, but Java 13 introduces multiline strings (finally). That might be a nice way to communicate what the expected response looks like if it is available.

Reply:
Will look into that

Reply:
I think I might be using Java 11